README.md

```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project
+
+> In this website you will learn how to create a web application using the Html Css and Javascript.
```

Comment:
- [ ] kindly add a good description of what your project is all about.  From the README the user will know what to expect